timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -62,8 +67,8 @@ def __init__(
         prediction_length: int = 1,
         path: Optional[str] = None,
         name: Optional[str] = None,
-        eval_metric: str = None,
-        hyperparameters: Dict[str, Any] = None,
+        eval_metric: Optional[str] = None,
```

Comment:
`eval_metric: Optional[Union[str, TimeSeriesScorer]]` for consistency with `AbstractTimeSeriesModel`?